README.md

```diff
@@ -1,2 +1,50 @@
 # PrabhakarAndAlex-Old-Apple.com-Clone
 This project is focused on learning gradient and background image positioning using CSS. In this project we're required to create clone of Old Apple.com's homepage.
+
+![screenshot](images/SiteScreenshot.png)
+
+In this task we were required to create a clone of old Apple.com's home page using the best practices of HTML AND CSS with GitHub process flow.
+
+## Built With
+
+- HTML,
+- CSS3,
+
+## Live Demo
+
+[Live Demo Link]()
```

Comment:
fix the live demo link you can use [githack](https://raw.githack.com/) for generating live demos.

Reply:
Adding it. Thank You!